Change

@package vaimo/composer-patches-target1
@depends !vaimo/composer-patches-target2:>=1.0.0

--- src/example.txt.org
+++ src/example.txt
@@ -5,7 +5,7 @@
 value5
 value6
 value7
-value8
+valueX
 value9
 value10
 value11
